src/root/port.c

```diff
@@ -335,12 +335,26 @@ int Port::stricmp(const char *s1, const char *s2)
 
 float Port::strtof(const char *p, char **endp)
 {
-    return static_cast<float>(::strtod(p, endp));
+    if(endp)
+        return static_cast<float>(::strtod(p, endp)); // does not set errno for underflows, but unused
+
+    _CRT_FLOAT flt;
+    int res = _atoflt(&flt, p);
```

Comment:
Sorry, I didn't have all VS version around to test. I verified in MSDN that the function are available in older versions, but didn't noticed the different prototype.
Strange, that they did not change `_atodbl`, but that could have raised some suspicion.